lib/tizen_tpk.dart

```diff
@@ -181,6 +181,9 @@ class TizenManifest {
   /// The application type (either "capp" or "dotnet").
   String? get applicationType => _applications.first.getAttribute('type');
 
+  /// The TizenFX API level of the application.
+  String? get dotnetApiLevel => _applications.first.getAttribute('api-version');
```

Comment:
Unfortunately,.. `api-version` attribute of `ui-applicatoin` has been deprecated since Tizen 6.5. So currently `tizen-manifest.xml` does not describe `C# API Level` anymore. But we can inter the `API Level` with `api-version` of `manifest` element by using a mapping table. 
refer: https://github.com/Samsung/Tizen.NET/issues/138

Reply:
How about just passing the platform version to the embedding.csproj? You may use the platform version in conditions.

Reply:
Fixed. Thank you for the information!